src/test/resources/db/migration/V002__Master_data.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO Product(modificationCounter, title, description, price) VALUES (1, 'Testproduct 1', 'Description of testproduct one', 1.99);
```

Comment:
You should not create test data in a versioned (V002) script as it will collide with productive scripts.
To add test data if you need, better use [test-data builders](https://github.com/devonfw/cobigen/wiki/cobigen-usecases#testdata-builder-for-devon4j) or if you really need an sql script, better go for flyway lifecycle hook [after-migrate](https://flywaydb.org/documentation/concepts/callbacks) to get rid of script versioning and let the test scripts just been applied at the very end of every startup

Reply:
I deleted the file. Now the database scripts in src/main/resources are also used for the tests